build-version.sh

```diff
@@ -57,6 +57,6 @@ done
 #
 # TODO: maybe we want to make a ~/.cargo-quickinstall/bin to untar into,
 # and add symlinks into ~/.cargo/bin, to aid debugging?
-tar -czf "${TEMPDIR}/${CRATE}-${VERSION}-${TARGET_ARCH}.tar.gz" $BINARIES
+tar --posix -czf "${TEMPDIR}/${CRATE}-${VERSION}-${TARGET_ARCH}.tar.gz" $BINARIES
```

Comment:
We might have to use `--portability` here:

```suggestion
tar --portability -czf "${TEMPDIR}/${CRATE}-${VERSION}-${TARGET_ARCH}.tar.gz" $BINARIES
```

Reply:
Amusingly, I couldn't see --portability in the bsdtar man page.

Reply:
Perhaps we can use `--format=v7` instead (`--portability` seems to be same as it)?